mw-webapp/src/logic/allUsersPage/AllUsersPage.tsx

```diff
@@ -1,17 +1,30 @@
 import {useState} from "react";
 import {Button, ButtonType} from "src/component/button/Button";
 import {HorizontalContainer} from "src/component/horizontalContainer/HorizontalContainer";
+import {Input} from "src/component/input/Input";
 import {Loader} from "src/component/loader/Loader";
 import {displayNotification} from "src/component/notification/displayNotification";
 import {ScrollableBlock} from "src/component/scrollableBlock/ScrollableBlock";
 import {HeadingLevel, Title} from "src/component/title/Title";
 import {UserPreviewDAL} from "src/dataAccessLogic/UserPreviewDAL";
 import {useLoad} from "src/hooks/useLoad";
+import {usePersistanceState} from "src/hooks/usePersistanceState";
 import {LAST_INDEX} from "src/logic/mathConstants";
 import {UsersTableBlock} from "src/logic/usersTable/UsersTableBlock";
 import {UserPreview} from "src/model/businessModelPreview/UserPreview";
+import {AllUsersPageSettings} from "src/utils/LocalStorageWorker";
 import styles from "src/logic/allUsersPage/AllUsersPage.module.scss";
 
+const FILTER_EMAIL_EMPTY_STRING = "";
+const DEFAULT_ALL_USERS_PAGE_SETTINGS: AllUsersPageSettings = {filterEmail: FILTER_EMAIL_EMPTY_STRING};
+
+/**
+ * Safe opened tab from localStorage
+ */
+const allUsersPageSettingsValidator = (currentSettings: AllUsersPageSettings) => {
+  return !!currentSettings.filterEmail;
```

Comment:
are you sure we should save filters in localstorage?

Reply:
it is strange validation

Reply:
Yea, deleted, thanks